src/libs/auth.ts

```diff
@@ -450,23 +453,28 @@ export const processUser = (user: OidcUser | null, isSignInEvent: boolean): void
   });
 };
 
-const initializeTermsOfService = (isSignedIn: boolean, state: AuthState): TermsOfServiceStatus => {
-  return {
-    userHasAcceptedLatestTos: isSignedIn ? state.termsOfService.userHasAcceptedLatestTos : undefined,
-    permitsSystemUsage: isSignedIn ? state.termsOfService.permitsSystemUsage : undefined,
-  };
-};
-
-export const initializeAuth = _.memoize(async () => {
```

Comment:
I didn't even realize this was here. Now I'm _really_ suspicious that `react-oidc-context` and AuthStoreSetter aren't necessary. But that can be a project for another day.